src/Sulu/Bundle/AdminBundle/Resources/js/stores/clipboardStore/clipboardStore.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import type {Observer} from './types';
+
+class ClipboardStore {
+    observers: {[key: string]: Array<Observer>} = {};
+    storageEventListener: ?(event: StorageEvent) => void;
+
+    updateStorageEventListener(): void {
+        const activeObservers = Object.values(this.observers).flat().length;
+
+        // listen for "storage" events to notify observers if something is copied in another browser window
+        if (activeObservers > 0 && !this.storageEventListener) {
+            this.storageEventListener = (event: StorageEvent) => {
+                if (event.key) {
+                    this.notifyObservers(event.key, event.newValue ? JSON.parse(event.newValue) : undefined);
+                }
+            };
+            window.addEventListener('storage', this.storageEventListener);
+        } else if (activeObservers === 0 && this.storageEventListener) {
+            window.removeEventListener('storage', this.storageEventListener);
+        }
+    }
+
+    notifyObservers(key: string, value: mixed): void {
+        const observers = this.observers[key] || [];
+
+        for (const observer of observers) {
+            observer(value);
+        }
+    }
+
+    get(key: string): mixed {
```

Comment:
I would like to avoid the `get` as it would make it possible to use then without a observer which should not be the case.